Docs: supported agencies, enrollment pathways

machikoyasuda opened the pull request:
closes #1791 

## What this PR does
- Adds table to show what transit agencies use what enrollment pathways, with year launched - in the Project overview section
- Adds table to show what enrollment pathways are available, and their general status - in the Project overview section
- Changes `Use cases` to `Enrollment pathways`
- Removes College discount
- Renames `Seniors` to `Older adults`

## Project overview
<img width="983" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/4a4c14d9-cadd-41a1-9156-8152cc795a68">

## Sidebar
<img width="242" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/72721593-5c49-4490-a91d-2912e279f7bc">

machikoyasuda commented:
@indexing What do you think?

<img width="1056" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/5beaffb3-0161-49de-a79b-0fc6abb34d4d">

In the future, when new enrollment pathways are planned out - we can add them and add it as a status of  "In contracting", "In product/design", "In development".

indexing commented:
@machikoyasuda I have a few small suggestions.
![Image Dec 12 at 11 AM](https://github.com/cal-itp/benefits/assets/6279581/cc79978c-4248-47e6-8e8e-bda9b48ea15f)

machikoyasuda commented:
@indexing 
<img width="1047" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/4a443d4a-c7ce-4d96-92d0-07a665ef2e89">

- Updated `in production validation` to just `in test` for SBMTD/Mobility Pass. I think `In test` could be used to describe the various states between `the code is done and deployed but we are waiting on agencies/permissions/contracts/politics`.
- Is 10/2023 launch for SBMTD/Older adults correct(??) I forgot.
- I added links to the specific release notes for each enrollment pathway launch.

indexing commented:
_The Cal-ITP Benefits app supports the following enrollment pathways that ~~uses~~ use the corresponding eligibility verification methods._

I note a small verb from typo. Otherwise, looks good! The left alignment of column content _and_ the heading is a cleaner look.

machikoyasuda commented:
@indexing fixed: https://github.com/cal-itp/benefits/pull/1815/commits/62e1b8f76a43946b68905a5fe357623efb94e6f5

indexing commented on docs/README.md:
```diff
@@ -18,6 +18,28 @@ The development of this publicly-accessible client is being managed by Caltrans'
+
+| Transit agency                                  | Older adults | Agency card | Veterans | Initial agency launch |
+| ----------------------------------------------- | ------------ | ----------- | -------- | --------------------- |
+| **Monterey-Salinas Transit**                    | Live         | Live        | Live     | 5/2021                |
```
A small formatting suggestion for styling and legibility: represent single months with two digits rather than one, i.e. 05 rather than 5. All date with read cleanly together.

machikoyasuda commented on docs/README.md:
```diff
@@ -18,6 +18,28 @@ The development of this publicly-accessible client is being managed by Caltrans'
+
+| Transit agency                                  | Older adults | Agency card | Veterans | Initial agency launch |
+| ----------------------------------------------- | ------------ | ----------- | -------- | --------------------- |
+| **Monterey-Salinas Transit**                    | Live         | Live        | Live     | 5/2021                |
```
```suggestion
| **Monterey-Salinas Transit**                    | Live         | Live        | Live     | 05/2021                |
```

machikoyasuda replied:
Resolved: https://github.com/cal-itp/benefits/pull/1815/commits/d1281e88476c816d90f805f65eb0d67ec44ff11c https://github.com/cal-itp/benefits/pull/1815/commits/4468bafce04f37baaab2b020ab184959854a428d

machikoyasuda commented on docs/README.md:
```diff
@@ -18,6 +18,28 @@ The development of this publicly-accessible client is being managed by Caltrans'
+
+| Enrollment pathway                                             | Eligibility verification                                                              | Status | Launch                                                                |
+| -------------------------------------------------------------- | ------------------------------------------------------------------------------------- | ------ | --------------------------------------------------------------------- |
+| [**Older adults**](/benefits/enrollment-pathways/older-adults) | [Login.gov IAL2](https://developers.login.gov/attributes/)                            | Live   | [8/2022](https://github.com/cal-itp/benefits/releases/tag/2022.08.1)  |
```
```suggestion
| [**Older adults**](/benefits/enrollment-pathways/older-adults) | [Login.gov IAL2](https://developers.login.gov/attributes/)                            | Live   | [08/2022](https://github.com/cal-itp/benefits/releases/tag/2022.08.1)  |
```

machikoyasuda commented on docs/README.md:
```diff
@@ -18,6 +18,28 @@ The development of this publicly-accessible client is being managed by Caltrans'
+| -------------------------------------------------------------- | ------------------------------------------------------------------------------------- | ------ | --------------------------------------------------------------------- |
+| [**Older adults**](/benefits/enrollment-pathways/older-adults) | [Login.gov IAL2](https://developers.login.gov/attributes/)                            | Live   | [08/2022](https://github.com/cal-itp/benefits/releases/tag/2022.08.1)  |
+| [**Agency cards**](/benefits/enrollment-pathways/agency-cards) | [Eligibility API](https://docs.calitp.org/eligibility-api/specification/)             | Live   | [11/2022](https://github.com/cal-itp/benefits/releases/tag/2022.11.1) |
+| [**Veterans**](/benefits/enrollment-pathways/veterans)         | [Veteran Confirmation API](https://developer.va.gov/explore/api/veteran-confirmation) | Live   | [9/2023](https://github.com/cal-itp/benefits/releases/tag/2023.09.1)  |
```
```suggestion
| [**Veterans**](/benefits/enrollment-pathways/veterans)         | [Veteran Confirmation API](https://developer.va.gov/explore/api/veteran-confirmation) | Live   | [09/2023](https://github.com/cal-itp/benefits/releases/tag/2023.09.1)  |
```

machikoyasuda commented:
Needs final review from @angela-tran (codeowner)

angela-tran commented on docs/README.md:
```diff
@@ -18,6 +18,28 @@ The development of this publicly-accessible client is being managed by Caltrans'
+| ----------------------------------------------- | ------------ | ----------- | -------- | --------------------- |
+| **Monterey-Salinas Transit**                    | Live         | Live        | Live     | 05/2021                |
+| **Santa Barbara Metropolitan Transit District** | Live         | In test     |          | 10/2023               |
+| **Sacramento Regional Transit District**        | In test      |             |          |                       |
```
I'm wondering if we might want to track more than just the initial launch date, as in track the dates that a specific enrollment pathway was launched for a specific transit agency. So the table would be more like:
`| Transit agency | Enrollment pathway | Launch date |`

I think what you have here is a good start though, so I'll leave this as a thought that can be a future PR if we want

machikoyasuda replied:
Yea we could but that's so granular, and it kinda really only affects MST - since most agencies don't offer a Veterans discount. We're trying to phase out Agency Cards, and perhaps in the future, more agencies can go live with multiple enrollment pathways so we don't have to be doing these incremental rollouts.
